documentation/asciidoc/topics/yaml/transport_encryption.yaml

```diff
@@ -0,0 +1,21 @@
+  infinispan:
+    jgroups:
+      stack:
```

Comment:
Do we need to define a dedicated stack or can we just do:

```yaml
deploy:
  infinispan:
    cacheContainer:
      transport:
        secuirty-realm="cluster-transport"
    server:
      security:
        securityRealms:
          - name: cluster-transport
            serverIdentities:
              ssl:
                keystore:
                  alias: "server"
                  path: "/etc/encrypt/transport/keystore.p12"
                  password: "password"
```

Reply:
from the discussion [here]|(https://issues.redhat.com/browse/DGSUP-257) seems that we need to extend "kubernetes" jgroups stack

Reply:
In the [Infinispan server docs](https://infinispan.org/docs/stable/titles/server/server.html#securing-cluster-transport-tls-identities_secure-cluster-transport) we recommend the security-realm approach so I think that's the recommended configuration. IMO this is better approach as it abstract the JGroups internal details from the user.

Maybe @tristantarrant has some thoughts as the SME.

Reply:
> In the [Infinispan server docs](https://infinispan.org/docs/stable/titles/server/server.html#securing-cluster-transport-tls-identities_secure-cluster-transport) we recommend the security-realm approach so I think that's the recommended configuration. IMO this is better approach as it abstract the JGroups internal details from the user.
> 
> Maybe @tristantarrant has some thoughts as the SME?

discussed with Tristan, I changed from jgroups to realm TLS